GitCommands/Git/SubmoduleHelpers.cs

```diff
@@ -1,46 +1,47 @@
 ﻿using System.Diagnostics.CodeAnalysis;
 using System.Text.RegularExpressions;
-using GitCommands.Patches;
 using GitUIPluginInterfaces;
 using Microsoft;
 
 namespace GitCommands.Git
 {
     public static class SubmoduleHelpers
     {
-        public static async Task<GitSubmoduleStatus?> GetCurrentSubmoduleChangesAsync(GitModule module, string? fileName, string? oldFileName, ObjectId? firstId, ObjectId? secondId)
+        public static async Task<GitSubmoduleStatus?> GetCurrentSubmoduleChangesAsync(IGitModule module, string? fileName, string? oldFileName, ObjectId? firstId, ObjectId? secondId)
         {
-            (Patch? patch, string? errorMessage) = await module.GetSingleDiffAsync(firstId, secondId, fileName, oldFileName, "", GitModule.SystemEncoding, cacheResult: true).ConfigureAwait(false);
+            (Patch? patch, string? errorMessage) =
+                await module.GetSingleDiffAsync(firstId, secondId, fileName, oldFileName, "", GitModule.SystemEncoding, cacheResult: true, isTracked: true)
+                            .ConfigureAwait(false);
```

Comment:
```suggestion
            (Patch? patch, string? errorMessage)
                = await module.GetSingleDiffAsync(firstId, secondId, fileName, oldFileName, "", GitModule.SystemEncoding, cacheResult: true, isTracked: true)
                    .ConfigureAwait(false);
```

Reply:
`=` is generally the last symbol when lines are wrapped:
![image](https://github.com/gitextensions/gitextensions/assets/4403806/4ff4bcf5-8c00-404a-ad44-2e7d940b177f)
![image](https://github.com/gitextensions/gitextensions/assets/4403806/6f7fbf35-3fe1-4347-8edf-471c214f756a)

Reply:
It's the new code style - supporting left reading.

Reply:
> ..supporting left reading

Could you please elaborate? I'm not sure what this means

Reply:
Hold your right hand over the code and try to get the main intention of the code.
With the next line starting with "=" or "." it is clear that it continues the previous line and how it does.

Reply:
I'm a left-hander so for me it doesn't work the same :)